spec/models/user_spec.rb

```diff
@@ -0,0 +1,75 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  before do
+    @user = FactoryBot.build(:user)
+  end
+
+  describe 'ユーザー新規登録' do
+    it 'nicknameが空では登録できない' do
```

Comment:
```suggestion
  describe 'ユーザー新規登録' do
    it 'nicknameが空では登録できない' do
```
過去アプリと同様にテストをcontextで正常系と異常系のグループ分けしましょう。